.github/workflows/ci-build.yml

```diff
@@ -59,9 +59,10 @@ jobs:
           - embroider-optimized
 
     steps:
-      - uses: actions/checkout@v3
-      - uses: volta-cli/action@v4
-      - name: install dependencies
-        run: npm install
+      - uses: actions/checkout@v4
+      - uses: wyvox/action-setup-pnpm@v3
+        with:
+          node-version: 18.18.1
```

Comment:
@NullVoxPopuli looks strange we specify Node version only in this job, is that miss or really necessary?

Reply:
it's been needed until default node is this specific patch, because 18.17 and earlier was successful with webpack, but 18.18, (and 20+) had an issue with libuv, https://github.com/nodejs/node/issues/49911

the specifying specifically, rather than relying on the volta config (or literally any other node version file), but action/setup-node (encapsulated by the wyvox action here) doesn't correctly support package.json as a "node version file" -- I've submitted a PR here: https://github.com/actions/setup-node/issues/864#issuecomment-1752169306 which should automatically propagate once merged and released, but it's been slow going getting anyone to look at (I don't really know who to ping)